### DISCUSSION TITLE
g and q looks almost same because they get cut in search bar

### DISCUSSION THREAD

**Discussion by heysujal** [pradyunsg/furo#666] [Potential Issue]
here are screenshots from different zoom level

![image](https://github.com/pradyunsg/furo/assets/55016909/01fd3af4-d681-429c-a904-41bd6af73b09)

![image](https://github.com/pradyunsg/furo/assets/55016909/57fcdd00-f406-4b4a-a8a2-3b5da10da9f6)

**Comment by pradyunsg** [comment]
Ooo. A few questions:

- What browser is this?
- What OS is this?
- What screen resolution is this with?
- What zoom levels is this with?

**Reply by heysujal** [reply]
Chrome 
Windows 11
1920*1080 
100% and 125%
I tried to fix it on my own but wasn't able to set the project up
